Tasks/PackerBuild/src/packerHost.ts

```diff
@@ -137,6 +138,23 @@ export default class PackerHost implements definitions.IPackerHost {
         return null;
     }
 
+    private async _waitForPackerExecutable(packerPath: string): Promise<void> {
+        if(!!packerPath && tl.exist(packerPath)) {
+            var iterationCount = 0;
+            do{
+                // query version to check if packer executable is ready
+                var result = tl.tool(packerPath).arg("--version").execSync();
+                if(result.code != 0 && result.error && result.error.message.indexOf("EBUSY") != -1){
```

Comment:
In case there is a failure; but the failure is not EBusy - should you fail the task or ignore and continue?